test/webdriverio/test/basic_test.ts

```diff
@@ -44,13 +44,12 @@ suite('Keyboard navigation on Blocks', function () {
       await this.browser.pause(PAUSE_TIME);
     }
 
-    const selectedId = await this.browser.execute(() => {
-      return Blockly.common.getSelected()?.id;
-    });
-    chai.assert.equal(selectedId, 'controls_repeat_1');
+    chai
+      .expect(await getCurrentFocusNodeId(this.browser))
+      .to.include('controls_if_2');
```

Comment:
I think this be an assert.equal here and elsewhere. The include is only needed for connections and fields because they get a generated string appended to their id, but blocks the whole id is known so we can use equal.

Reply:
Done.